compose.js

```diff
@@ -88,12 +92,10 @@ function mergeComposable(dstDescriptor, srcComposable) {
   combineProperty('configuration', assign);
   combineProperty('deepConfiguration', merge);
   if (Array.isArray(srcDescriptor.initializers)) {
-    dstDescriptor.initializers = srcDescriptor.initializers.reduce((result, init) => {
-      if (isFunction(init) && result.indexOf(init) < 0) {
-        result.push(init);
-      }
-      return result;
-    }, Array.isArray(dstDescriptor.initializers) ? dstDescriptor.initializers : []);
+    const initializers = (dstDescriptor.initializers || [])
```

Comment:
The "completely internal" part is wrong. Stamps, actually, allow and encourage people to manipulate the descriptor (aka metadata). We do not limit people in what they do with their own code. So, our code should tolerate with all kind of rubbish stored in the descriptor.
https://github.com/stampit-org/check-compose/blob/master/src/initializer-tests.js#L240

Does that answer your question?